src-electron/zcl/zcl-loader-silabs.js

```diff
@@ -1591,6 +1591,9 @@ function prepareDeviceType(deviceType) {
     scope: deviceType.scope ? deviceType.scope[0] : '',
     superset: deviceType.superset ? deviceType.superset[0] : '',
   }
+  if ('endpointComposition' in deviceType) {
+    ret.composition = deviceType.endpointComposition[0].type[0]
```

Comment:
Will endpoint composition always have just one composition with 1 type i.e. either tree or full-family associated with it?
I see other xml tags what happens to those with the endpointComposition?
`<endpoint mandatory ="true">
                   <deviceType>0x0071</deviceType>
           </endpoint>`

Reply:
Yes